src/types/widget.ts

```diff
@@ -118,9 +118,6 @@ export type BuiltinWidgetTypes =
   | 'ais.toggleRefinement'
   | 'ais.voiceSearch';
 
-// In this case we don't want to allow any record, as it would no longer warn
-// for unexpected parameters
-// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
do you know why this no longer errors for `object` usage?

Reply:
It looks like `object` is considered safe now, and `Object` is banned (see [rule details](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-types.md#rule-details)).